src/main/java/org/isf/patient/manager/PatientBrowserManager.java

```diff
@@ -21,11 +21,6 @@
  */
 package org.isf.patient.manager;
 
-import java.util.ArrayList;
```

Comment:
Please see the **Code style** section of the repo README file to enable the project's code style rules.  For example, package imports are not supported and import ordering is

- statics
- java
- javax
- org
- com

Thanks.

Reply:
Sorry, I thought I had the correct settings.
Now it should be ok.